apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -981,8 +981,9 @@ const Ticket = new GQLListSchema('Ticket', {
             },
             {
                 type: 'BTreeIndex',
-                fields: ['assignee', 'executor', 'organization'],
-                name: 'ticket_assignee_exec_org',
+                fields: [ 'organization', 'assignee', 'executor'],
+                name: 'ticket_org_assignee_exec',
+                condition: 'Q(deletedAt__isnull=True)',
```

Comment:
Should we use `deletedAt`?

Reply:
i removed the filter and added "deletedAt" to the fields -- the query still uses the index, the cost is the same as with the filter, but i think it could benefit all the queries that do not use deletedAt=null filter